templates/podsecuritypolicy.yaml

```diff
@@ -0,0 +1,60 @@
+{{- if not .Values.csi.enable }}
```

Comment:
For CSI, I think you would just need to bind the role to different service account, i.e. `{{ template "storageos.fullname" . }}-daemonset-sa`. So, you can move this conditional below where you define the role binding subject.

Correct me if I'm wrong. Pods that don't need any of the special permission, like the pods in statefulset, need not have pod-security-policy attached to them right?